force-app/main/default/lwc/scopedNotification/scopedNotification.js

```diff
@@ -36,6 +37,10 @@ export default class ScopedNotification extends LightningElement {
             slds += `slds-theme_${theme}`;
         }
 
+        if (this.rounded) {
+            slds += " slds-box";
+        }
+
```

Comment:
is this part of the scoped notif spec?

Reply:
I don't think so, UX wanted rounded corners.